mesop/components/autocomplete/autocomplete.ng.html

```diff
@@ -0,0 +1,40 @@
+<mat-form-field
+  [hideRequiredMarker]="config().getHideRequiredMarker()"
+  [color]="getColor()"
+  [floatLabel]="getFloatLabel()"
+  [appearance]="getAppearance()"
+  [subscriptSizing]="getSubscriptSizing()"
+  [hintLabel]="config().getHintLabel()!"
+  [style]="getStyle()"
+>
+  <mat-label *ngIf="config().getLabel()">{{config().getLabel()}}</mat-label>
+  <input
+    type="text"
+    [placeholder]="config().getPlaceholder()!"
+    matInput
+    (input)="onInput($event)"
+    (keyup)="onKeyUp($event)"
+    [formControl]="autocompleteControl"
+    [matAutocomplete]="auto"
+  />
+  <mat-autocomplete
+    (optionSelected)="onAutocompleteSelectEvent($event)"
+    #auto="matAutocomplete"
+  >
+    @for (option of filteredOptions | async; track $index) { @if
+    (option.getOptionGroup()) {
```

Comment:
this formatting looks a bit funky, not sure if prettier is forcing it?

Reply:
Hmm, yeah looks like the pre-commit is updating it to look like that. I tried reformatting, but after commiting it changed to that format again.

Reply:
ok, i think prettier doesn't quite understand angular's new-ish syntax @for / @if